src/app/app.component.html

```diff
@@ -71,39 +71,43 @@
               mat-button color="accent">#BlackLivesMatter</a>
           </span>
           <span class="d-none d-md-inline">
-            <button mat-button class="nav-button" routerLink="home" routerLinkActive="active">
+            <a mat-button class="nav-button" routerLink="home" routerLinkActive="active">
               <mat-icon>home</mat-icon>
               <span class="d-none d-xl-inline"> Home</span>
-            </button>
-            <button mat-button class="nav-button" routerLink="gear" routerLinkActive="active">
+            </a>
+            <a mat-button class="nav-button" routerLink="gear" routerLinkActive="active">
               <fa-icon [icon]="iconService.fasBowArrow"></fa-icon>
               <span class="d-none d-xl-inline"> Gear</span>
-            </button>
-            <button mat-button class="nav-button" routerLink="friends" routerLinkActive="active">
+            </a>
+            <a *ngIf="debugmode | async" mat-button class="nav-button" routerLink="todo-list" routerLinkActive="active">
```

Comment:
New page added here. It's hidden by default, shown if debug mode is on. The page itself is fine to go to. I plan on adding some route-guards/resolvers to prevent people from coming to the page if they aren't logged in.